src/ui_basic/table.h

```diff
@@ -135,6 +135,17 @@ template <> class Table<void*> : public Panel {
 		void set_string(uint8_t col, const std::string& str);
 		const Image* get_picture(uint8_t col) const;
 		const std::string& get_string(uint8_t col) const;
+
+		/*
+		 * The special "for_sorting" value allows overriding the default sort order.
+		 * This is the minimal-invasive fix for the bug that in English,
+		 * "Won" sorts after "Lost", while in German, "Gewonnen" sorts before
+		 * "Verloren", but we want "Won" to sort before "Lost" in all languages.
+		 * This override is only respected by the default column compare.
+		 */
+		void set_for_sorting(uint8_t col, int32_t value);
```

Comment:
This is not needed. Have a look at `FullscreenMenuMapSelect::compare_players` to see how the Table class supports comparators already.

Reply:
*Headdesk*
Will remove the new functionality and use the existing instead :P